db/schema.rb

```diff
@@ -12,6 +12,9 @@
 
 ActiveRecord::Schema.define(version: 2019_03_13_201707) do
 
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
```

Comment:
Was this automatically added by mistake or is it really required?

Reply:
Well...we are using postgres as the DB...and we should have that in out schema.